core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -664,13 +669,10 @@
       }
 
       .conversation-skin-supplemental-card-container {
-        left: 50%;
```

Comment:
Hi @maitbayev, when you do come across it, please do create a new issue. (I did a test on narrow screens (and mobile) and the helper card rendered fine).